www/md_base/src/styles/about.less

```diff
@@ -0,0 +1,35 @@
+.about{
+  padding: 20px;
```

Comment:
The styles for section will be moved to be a general style as I think it could be used at settings page too. 

Will make this change soon :)